contracts/committee/Committee.sol

```diff
@@ -259,7 +259,7 @@ contract Committee is MasterCopyNonUpgradable, ConsensusModule, CommitteeI {
             "Proposal must not be zero."
         );
 
-        consensus = ConsensusI(_consensus);
+        setupConsensus(_consensus);
```

Comment:
We can place this function call on line 240.

Reply:
It is not possible for functions.
In case constructor,it is possible to call base class constructor.